pkg/embeddedyamls/yamls.go

```diff
@@ -2837,14 +2837,6 @@ rules:
     verbs:
       - get
       - list
-  - apiGroups:  # pods are created to run firewall diagnostics
```

Comment:
This file is auto generated, you should be editing the source file `config/rbac/submariner-diagnose/cluster_role.yaml`